TESTS.md

```diff
@@ -0,0 +1,84 @@
+# Test guidlines
```

Comment:
Tiny typo: should be "Guidelines". Let's also use title case here (capital "G").